### PR TITLE
fix: common checks

### PR DESCRIPTION
**Pull request by gauravlochab**
This has code cleanup and changes for common  checks

### CODE REVIEW

File: operate/cli.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #   limitations under the License.
 #
 # ------------------------------------------------------------------------------
-
+# type: ignore
```
Comment (Adamantios) [inline operate/cli.py:19]: Why?

File: operate/services/service.py [hunk 2]
```diff
@@ -377,10 +376,11 @@ def load(cls, path: Path) -> "Deployment":
         """Load a service"""
         return super().load(path)  # type: ignore
 
+    # pylint: disable=too-many-locals
     def _build_docker(
         self,
         force: bool = True,
-        chain_id: str = "100",
+        chain_id: str = "100",  # pylint: disable=unused-argument
```
Comment (Adamantios) [inline operate/services/service.py:383]: Why disable and not remove instead?

File: operate/services/protocol.py [hunk 3]
```diff
@@ -17,6 +17,7 @@
 #   limitations under the License.
 #
 # ------------------------------------------------------------------------------
+# type: ignore
 """This module implements the onchain manager."""
```
Comment (Adamantios) [inline operate/services/protocol.py:21]: Why ignore the whole file?

File: operate/cli.py [hunk 4]
```diff
@@ -650,7 +656,7 @@ async def _deploy_service_onchain(request: Request) -> JSONResponse:
         operate.service_manager().deploy_service_onchain(
             hash=request.path_params["service"]
         )
-        operate.service_manager().stake_service_on_chain(
+        operate.service_manager().stake_service_on_chain(  # pylint: disable=no-value-for-parameter
```
Comment (Adamantios) [inline operate/cli.py:659]: The definition seems to require more args:

https://github.com/valory-xyz/optimus-quickstart/blob/bec0de58048f7712e30fccaa2bdc0530ad4dc14c/operate/services/manage.py#L888

File: operate/services/manage.py [hunk 5]
```diff
@@ -17,6 +17,7 @@
 #   limitations under the License.
 #
 # ------------------------------------------------------------------------------
+# type: ignore
```
Comment (Adamantios) [inline operate/services/manage.py:20]: Same question here.

File: operate/services/service.py (hunk 2 above)
Comment (gauravlochab) [reply operate/services/service.py:383]: removed in next push

File: operate/cli.py (hunk 4 above)
Comment (gauravlochab) [reply operate/cli.py]: the endpoint not used yet and the [function](https://github.com/valory-xyz/optimus-quickstart/blob/bec0de58048f7712e30fccaa2bdc0530ad4dc14c/operate/services/manage.py#L888)  in not implemented

File: operate/services/service.py (hunk 2 above)
Comment (Adamantios) [reply operate/services/service.py]: Hmm I still cannot see where the `_chain_id` is used in the method.

**Review by Adamantios — APPROVED** [review]

**Pull request merged**